javers-core/src/main/java/org/javers/common/reflection/JaversGetterFactory.java

```diff
@@ -42,6 +42,7 @@ private void findAllGetters(Class currentGetterSource) {
                     .filter(method -> isGetter(method) && !method.isBridge())
                     .filter(method -> !isOverridden(method, getters))
                     .map(getter -> createJaversGetter(getter, context))
+                    .filter(getter -> excludeDuplicatedProperties(getter, getters))
```

Comment:
are we sure that filter() for second getter will be executed after getters::add for first getter?

Reply:
`isOverriden` works the same way

Reply:
right :)